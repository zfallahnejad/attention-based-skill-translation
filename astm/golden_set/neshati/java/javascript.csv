157882,javascript
573032,javascript
182668,javascript
304,javascript
116472,javascript
571407,javascript
203907,javascript
157247,javascript
3218114,javascript
181497,javascript
280410,javascript
1065197,javascript
114251,javascript
617373,javascript
