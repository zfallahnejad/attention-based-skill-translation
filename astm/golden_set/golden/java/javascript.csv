157882,111,68,0.6201117318435754
573032,26,25,0.5098039215686274
182668,22,19,0.5365853658536586
304,21,11,0.65625
116472,20,17,0.5405405405405406
571407,19,22,0.4634146341463415
203907,19,22,0.4634146341463415
157247,17,7,0.7083333333333334
3218114,16,16,0.5
181497,13,2,0.8666666666666667
280410,13,15,0.4642857142857143
1065197,12,11,0.5217391304347826
114251,11,2,0.8461538461538461
617373,11,2,0.8461538461538461
